pkgs/development/python-modules/dohq-artifactory/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+, requests
+, python-dateutil
+, pyjwt
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "dohq-artifactory";
+  version = "0.10.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-YBjIouC9yRK3N5CfHfqI62gUWCLhrWPWutzAkxg6NlM=";
+  };
+
+  pyproject = true;
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    requests
+    python-dateutil
+    pyjwt
+  ];
+
+  pythonImportsCheck = [ "artifactory" ];
+
+  pytestFlagsArray = [ "tests/unit" ];
```

Comment:
```suggestion
  nativeCheckInputs = [ pytestCheckHook ];

  pytestFlagsArray = [ "tests/unit" ];
```

Reply:
Seems like their tests are not published in the pypi package. Would it be better to fetch from GitHub, if only to run the tests?

Reply:
If we could get the source from github, please do so.